core/src/main/java/io/questdb/cutlass/line/tcp/LineWalAppender.java

```diff
@@ -332,6 +342,12 @@ private void appendToWal0(
                                         throw castError(tud.getTableNameUtf16(), "STRING", colType, ent.getName());
                                     }
                                     break;
+                                case ColumnType.LONG256:
+                                    CharSequence cs = entityValue.asAsciiCharSequence();
+                                    if (Numbers.extractLong256(cs, long256)) {
+                                        r.putLong256(columnIndex, cs);
```

Comment:
putLong256() sig supports log256 as the input, there is no need to parse string twice